src/lib.rs

```diff
@@ -1280,7 +1280,7 @@ impl ExpressionStatus {
     }
 }
 
-mod errors {
+pub mod errors {
```

Comment:
Actually you're right. I made a mistake when importing `duct::errors::Error` into my own scope. The only thing missing then is releasing a new version of duct.rs. 🎉